scripts/demo_vggt.py

```diff
@@ -113,6 +113,125 @@ def parse_args():
     )
     return parser.parse_args()
 
+def _build_pycolmap_intri(fidx, intrinsics, camera_type, extra_params=None):
+    """
+    Helper function to get camera parameters based on camera type.
+
+    Args:
+        fidx: Frame index
+        intrinsics: Camera intrinsic parameters
+        camera_type: Type of camera model
+        extra_params: Additional parameters for certain camera types
+
+    Returns:
+        pycolmap_intri: NumPy array of camera parameters
+    """
+    if camera_type == "PINHOLE":

```

Comment:
We have similar code in the repo already. Could you take a look? If we can use that that'd be better. If not, this PR is ready to be merged I think.